### DISCUSSION TITLE
Tab for current target info

### DISCUSSION THREAD

**Discussion by Dan-De-Sousa** [iaincollins/icarus#19] [Ideas & Suggestions]
This terminal is fantastic. I think a tab should be dedicated for the current target information. (Systems, Star, Planet, and NPC info)

**Comment by iaincollins** [comment]
Oh I would love that! Unfortunately I don't think it's possible to get that using the current API from the game. :-(

I have drawn a line at trying to extract data at runtime except via the API, as it would be fragile and also Frontier might take a dim view of it. Most developers do, as it tends to lead to people trying to things like people trying to write bots, although there are entirely innocent uses within the spirit of good third party tools.

However this is a feature I'd love to have. It's worth asking the developers about on the official forums, I would implemented it in the UI immediately if I could!

As they probably wouldn't want to fill up the logs with Target events, I think they would want to handle it the way CMDR and Ship status events are handled, and have a single file (e.g. `Target.json`) that they write to on a tick periodically (e.g. once a second) which is good enough for most third party tool use cases.

**Reply by excalith** [reply]
Just for heads up: Regarding the target NPC info (or player target) I answered what can be done on [issue 67](https://github.com/iaincollins/icarus/issues/67)
